.clangd

```diff
@@ -1,3 +1,6 @@
+CompileFlags:
+  Add: [-D__cpp_concepts=202002]
```

Comment:
Without this, `clangd` will not work correctly atm.
Maybe this will become redundant later when we change the way `gcc-12` and `clang` are added to our CI image.